### PR TITLE
Add editing protocols, creating/downloading/updating protocol documents.

### PR DESCRIPTION
**Pull request by deiferni**
Also no longer display links to edit protocols/preprotocols in all states.

### CODE REVIEW

File: opengever/meeting/browser/meetings/meeting.py [hunk 1]
```diff
@@ -143,6 +155,18 @@ def url_generate_pre_protocol(self):
     def url_download_pre_protocol(self):
         return DownloadGeneratedPreProtocol.url_for(self.context, self.model)
 
+    def url_protocol(self):
+        return EditPreProtocol.url_for(self.context, self.model)
```
Comment (phgross) [inline opengever/meeting/browser/meetings/meeting.py]: should it not be 
```python
return EditProtocol.url_for(self.context, self.model)
```

File: opengever/meeting/model/meeting.py [hunk 2]
```diff
@@ -107,6 +116,17 @@ def get_pre_protocol_filename(self):
             translate(_("Pre-Protocol"), context=getRequest()),
             normalizer.normalize(self.get_title()))
 
+    def get_protocol_title(self):
+        return u"{}-{}.docx".format(
```
Comment (phgross) [inline opengever/meeting/model/meeting.py]: The `title` should not return `docx` isn't it?

File: opengever/meeting/tests/test_protocol.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    def test_protocol_participants_can_be_edited(self, browser):
+        yesterday = date.today() - timedelta(days=1)
+        tomorrow = date.today() + timedelta(days=1)
+        peter = create(Builder('member'))
+        hans = create(Builder('member').having(
+            firstname=u'Hans', lastname=u'M\xfcller'))
+        create(Builder('membership').having(
+            member=peter,
+            committee=self.committee_model,
+            date_from=yesterday, date_to=tomorrow))
```
Comment (phgross) [inline opengever/meeting/tests/test_protocol.py]: could we add a helper (`as_active`) or default values to the membership builder, which set `date_from` to yesterday and `date_to` to tomorrow?

Comment (deiferni) [inline opengever/meeting/tests/test_protocol.py]: i like it! done.

File: opengever/meeting/browser/meetings/meeting.py (hunk 1 above)
Comment (deiferni) [inline opengever/meeting/browser/meetings/meeting.py]: nice catch :astonished:

File: opengever/meeting/model/meeting.py (hunk 2 above)
Comment (deiferni) [inline opengever/meeting/model/meeting.py]: changed

**Comment by deiferni** [comment]
@phgross updated

**Pull request merged**